lib/smart_proxy_main.rb

```diff
@@ -9,20 +9,22 @@
 require 'webrick/https'
 require 'daemon'
 
-require 'checks'
 require 'proxy/log'
 require 'proxy/settings'
 require 'proxy/settings/plugin'
 require 'proxy/settings/global'
 require 'proxy/dependency_injection'
+require 'proxy/default_di_wirings'
```

Comment:
Perhaps require this only from lib/proxy/plugin_initializer.rb, which is where it's used, rather than at the app level.  Probably the same for default plugin validators?